packages/models/test/fixtures/categories.mock.ts

```diff
@@ -67,5 +67,5 @@ export function categoryConfig(
 }
 
 export function categoryConfigs(): CategoryConfig[] {
-  return Object.values(CATEGORIES_MAP);
+  return Object.values(JSON.parse(JSON.stringify(CATEGORIES_MAP)));
```

Comment:
Why? :raised_eyebrow:

Reply:
I had mutation bugs

Reply:
Well, that's why it's better to never mutate :sweat_smile:

For production code these additional JSON parse/serialize operations would concern me, but since this is only for test, it not a problem, just quite confusing :man_shrugging: